OpenSource/checklists/01-set-up.md

```diff
@@ -97,7 +70,6 @@
 - [ ] Release process is automated and documented
```

Comment:
In Testing section, there's a typo in several bullets -> `Repository include**s**` instead of `include`